imports/ui/Accordian.jsx

```diff
@@ -9,50 +9,50 @@ export default class Accordian extends Component{
   getRandNum(){
     return Math.floor((Math.random() * 1000000) + 0);
   }
-  render(){
-    let accId = "accordion"+this.getRandNum();
-    let headingOneId = "headingOne"+this.getRandNum();
-    let collapseOneId = "collapseOne"+this.getRandNum();
+    render(){
+      let accId = "accordion"+this.getRandNum();
+      let headingOneId = "headingOne"+this.getRandNum();
+      let collapseOneId = "collapseOne"+this.getRandNum();
```

Comment:
The indentation looks messed up. Your indentation should be consistent with the rest of the code to avoid large diff and merge conflicts. Similar problem below.